crates/sui-types/src/execution.rs

```diff
@@ -205,6 +218,8 @@ impl Value {
                 },
                 _,
             ) => *used_in_non_entry_move_call,
+            // TODO(tzakian): Need to figure this out.
```

Comment:
The only thing you can do with a `Receiver<T>` is consume it to receive the object, right? So it doesn't seem possible to use the one in a non-entry move call and then have the value live to be queried about it, which implies `false` is the correct return value here.